### ISSUE TITLE
[Question] mining with FTB Ultimine(and other veinminer style mod) mod was effect get Trinkets?

### ISSUE THREAD

**Issue by chironader** [owmii/Lost-Trinkets#53]
mining with FTB Ultimine(and other veinminer style mod) mod was effect get Trinkets?
I wondered that it~!
 
and willing add effecting time(grow corp ore,Smelting fester) or effect like bone meal trinket?
or already add?

thank you make this wonderful mod~!

**Comment by BlueAgent** [comment]
Yes, you can get trinkets from anything that causes a BlockBreak event with a player. FTB Ultimine included it seems.

There's two Trinket requests here it seems?
1. Speed up crop growth
2. Speed up smelting

Probably something for Owmii to decide.
